### PR TITLE
Use Activity to initialize Google Map

### PR DESCRIPTION
**Pull request by alvinthen**
Update AirMapManager to use Activity obtained from ReactContextBaseJavaModule (part of update from [RN v0.29.0](https://github.com/facebook/react-native/releases/tag/v0.29.0))

This might address issues like #165 

Note: Not tested for backward compatibility

### CODE REVIEW

**Comment by jeveloper** [comment]
This should resolve that problem of using multiple mapviews

File: android/lib/src/main/java/com/airbnb/android/react/maps/AirMapManager.java [hunk 1]
```diff
@@ -57,7 +58,7 @@ protected AirMapView createViewInstance(ThemedReactContext context) {
         reactContext = context;
 
         try {
-            MapsInitializer.initialize(this.appContext);
+            MapsInitializer.initialize(new AirMapModule(this.appContext).getActivity());
```
Comment (felipecsl) [inline android/lib/src/main/java/com/airbnb/android/react/maps/AirMapManager.java]: nice hack

**Comment by felipecsl** [comment]
lgtm. needs to be rebased though

**Comment by jeveloper** [comment]
hey @spikebrehm  appreciate for your push on 0.8 :)

Mind looking into publishing this change

https://github.com/lelandrichardson/react-native-maps/issues/453

It's a case where there is more than one mapview and one of the devs tested out a fix.

Thank you

**Comment by spikebrehm** [comment]
Great, we're happy to merge this, it just needs a rebase. Ping @alvinthen

**Comment by alvinthen** [comment]
Updated with rebased branch

**Comment by spikebrehm** [comment]
Thanks!

**Comment by jeveloper** [comment]
Thanks @spikebrehm  Thats awesome

**Comment by rops** [comment]
I think this change is conflicting with https://github.com/lelandrichardson/react-native-maps/commit/e77765d91ece5f36073907fe25725770a6972dd7 , as `lib` was removed, together with its gradle files. Either need to move this into main src or restore gradle files for `lib`

**Comment by spikebrehm** [comment]
@rops you're right! I'll fix it.

**Comment by spikebrehm** [comment]
@rops the fix is here: https://github.com/lelandrichardson/react-native-maps/pull/526

**Comment by spikebrehm** [comment]
Just published [0.8.1](https://github.com/lelandrichardson/react-native-maps/releases/tag/v0.8.1) that includes this change.

**Comment by spikebrehm** [comment]
It turns out 0.8.1 was faulty, because my working directory had an additional example app containing the `react-native` node module, which got published as part of the NPM package (:facepalm:), causing a `@providesModule` error, so I've unpublished it and published [0.8.2](https://github.com/lelandrichardson/react-native-maps/releases/tag/v0.8.2).  Use that instead!

**Pull request merged**